.craft.yml

```diff
@@ -9,8 +9,8 @@ artifactProvider:
   name: github
 targets:
   - name: upm
-    releaseOwner: "getsentry"
-    releaseRepo: "unity"
+    releaseRepoOwner: "getsentry"
```

Comment:
Should this land on another PR? Or just push to main if we need this to release once craft is out.